azext_iot/central/params.py

```diff
@@ -106,12 +138,14 @@ def load_central_arguments(self, _):
         context.argument(
             "interface_id",
             options_list=["--interface-id", "-i"],
-            help="Interface name as specified in the device template. Example: c2dTestingTemplate_356",
+            help="The name of the interface as specified in the device template. You can find it by navigating to Device"
```

Comment:
we should link an aka.ms link showing how to get the interface-id

Reply:
Will work PM for this update later.